metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -393,6 +387,10 @@ def _get_exported_bigquery_audit_metadata(
         self, bigquery_client: BigQueryClient
     ) -> Iterable[BigQueryAuditMetadata]:
         if self.config.bigquery_audit_metadata_datasets is None:
+            self.error(
+                logger, "audit-metadata", "bigquery_audit_metadata_datasets missing"
```

Comment:
Why is this an error? It is an optional config param. Please replace `missing` both from the error message and the `self.report.bigquery_audit_metadata_datasets_missing` with `not_set`.

Reply:
Because if we are using exported audit metadata then you must have `bigquery_audit_metadata_datasets` otherwise there will be nothing ingested.

Reply:
Why don't we make this mandatory at the config-level itself then by making self.config.bigquery_audit_metadata_datasets not optional?